### PR TITLE
Help geoidesic move commands to property

### PR DESCRIPTION
**Pull request by jadb**
Supersedes #647

### CODE REVIEW

**Comment by geoidesic** [comment]
Hi. I managed to fix the failing tests as per AdMad's comments. I'll resubmit a PR.

**Comment by jadb** [comment]
@geoidesic, this PR also keeps the order of operations the same as it was instead of inverting what to loop through first (tables / commands).

File: src/Command/AllCommand.php [hunk 1]
```diff
@@ -82,11 +93,8 @@ public function execute(Arguments $args, ConsoleIo $io): ?int
             $tables = [$name];
         }
 
-        $commands = [
-            new ModelCommand(),
-            new ControllerCommand(),
-            new TemplateCommand(),
-        ];
+        $commands = array_map(function ($c) { return new $c; }, $this->commands);
```
Comment (dereuromark) [inline src/Command/AllCommand.php]: $command is better :)

Comment (dereuromark) [reply src/Command/AllCommand.php]: And inline annotation is still useful here.

**Comment by geoidesic** [comment]
> @geoidesic, this PR also keeps the order of operations the same as it was instead of inverting what to loop through first (tables / commands).

@jadb `array_map` introduces an extra loop, which isn't as performant or as concise as the flip. Not sure why this is better?

**Comment by dereuromark** [comment]
@geoidesic Performance on this nano level is not an issue here, especially not for CLI.
The main focus must be readability for developers.
If that answer is yes for the map, then OK. otherwise it should stay simple loop :)

**Comment by jadb** [comment]
@geoidesic, my goal was not to replace your PR which introduced a **good change**, but to extend on it as it was faster than comments back and forth. Never said one is better than the other, I just pointed out the difference between the two. 

I personally (doesn't mean am right) think that keeping things in the right order of operation (the one already defined) is more important than performance when using bake (only in development).

**Comment by geoidesic** [comment]
@jadb Ok sure. I don't know if changing the order could have a BC impact. It's possible I guess. So maybe your way is safer. I've submitted a new PR to master as there isn't a `2.next` branch. I don't mind how this gets refactored or pulled, just so long as the property is available :)

**Comment by dereuromark** [comment]
Other PR was merged. Follow up PRs can be done if needed.

**Pull request closed without merging**